Minesweeper.py

```diff
@@ -478,27 +469,22 @@ def clear_adjacent_tiles(self, row, column): # clears tiles adjacent to 0 tiles
                 ):
                     self.tile_state[r][c]['covered'] = False
                     if self.tile_state[r][c]['adjacent_mines'] == 0:
-                        self.clear_adjacent_tiles(r, c) # Recursively clear adjacent tiles
+                        self.clear_adjacent_tiles(r, c)
 
-# Define the Mouse class to handle mouse-related functionality
-class Mouse(object):
-    def __init__(self):
-        # DHB: this class is only being use as a side-effect of initialising it. Either use it properly, or remove it, and move the init code elsewhere
-        pygame.mouse.set_visible(True)
-        pygame.mouse.set_cursor(*pygame.cursors.tri_left)
+"""Create the game window"""
```

Comment:
this method of commenting is only used for classes and functions. Use # for normal inline comments.